pkg/tools/glustercli-auth-header.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from argparse import ArgumentParser
+from datetime import datetime, timedelta
+
+import jwt
+
+
+def jwt_token(user, secret):
+    claims = dict()
+    claims['iss'] = user
+    claims['iat'] = datetime.utcnow()
+    claims['exp'] = datetime.utcnow() + timedelta(seconds=10)
+
+    token = jwt.encode(claims, secret, algorithm='HS256')
```

Comment:
This looks fine, To provide more security, we need to add URL tampering protection also
@aravindavk any thoughts on  this

in the server side, we need to enable (HTTP method) validation to avoid miss use of the token.

Reply:
makes sense. Please open new issue for the same.